crate/README-short.txt

```diff
@@ -1 +1,2 @@
-Crate is an open source, highly scalable, shared-nothing distributed SQL database.
+CrateDB is a distributed SQL database that makes it simple to store and analyze
+massive amounts of machine data in real-time.
```

Comment:
This needs to be a single line, and needs to be less than 100 characters (as seen in the Travis output).